pkg/applications/providers/util/util.go

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
Overall I thinking if this needs to be its own util package.

I am saying this because:
a) 2 of 3 util-funcs are directly related to helm and I think should be part of the the helm providers (I would also be fine with code-duplication if they are needed in both Template and Sourceprovider
b) StatusUpdate func in my opinion could also become part of providers directly

But this is subjective I think, since we don't have a unique coding-style within the project. My comment is more meant as food-for-thought :)

Reply:
i initially put it in providers package but it create a cycle dependency between provider and template pakcages